routes/api/comments.js

```diff
@@ -6,42 +6,56 @@ const createError = require('http-errors');
 const router = express.Router();
 const { sequelize, CommentLike } = require("../../db/models");
 
-
-
-//Like a comment
+/* Like a comment */
 router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) => {
 
     if (!res.locals.authenticated || res.locals.user.id !== like.userId) {
+        /* Return error 401, if
+               user is not authenticated, or
+               user is not liker */
         return next(createError(401)); 
-    } else if (res.locals.authenticated ) {
-    
-    const likedComment = await CommentLike.create({
-            userId: res.locals.user.id,
-            commentId: req.params.id
-    })
-  }
-            res.status(201).json(likedComment);
-  }));
 
-
-//Unlike a comment
-router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
-    const like = CommentLike.findByPk(req.params.id);
-
-    if (!like) {res.status(404)};
+    } else if (res.locals.authenticated ) {
         
-    if (!res.locals.authenticated || res.locals.user.id !== like.userId) {
-        return next(createError(401)); 
-
-    } else if (res.locals.user.id === like.userId && res.locals.authenticated ) {
-             await CommentLike.destroy({where: {id:req.params.id}})
+        /* Create comment like */
+        const likedComment = await CommentLike.create({
+                userId: res.locals.user.id,
+                commentId: req.params.id
+            })
         }
-    
-                res.json({message: "Comment deleted"});
 
-}))
+        /* Set status 201, and
+           respond with json, likedComment */
+        res.status(201).json(likedComment);
+    }));
 
 
+/* Unlike a comment */
+router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
+
+        /* Initialize like object */
+        const like = CommentLike.findByPk(req.params.id);
```

Comment:
`CommentLike.findByPk(req.params.id);` returns a `Promise`. You'll need to await the promise so like is an instance of a CommentLike

Reply:
`await` keyword added to line 37.

Reply:
wait i thought i pushed these changes earlier lol